keyboards/heliar/wm1_hotswap/wm1.c

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2019 HELIAR MK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "wm1.h"
+
+
+void keyboard_pre_init_kb(void)
+{
+    setPinOutput(D7);
+    setPinOutput(D6);
+    setPinOutput(D4);
+}
+
+bool led_update_kb(led_t led_state) {
+    if (led_state.num_lock) {
+        writePinLow(D7);
+    } else {
+        writePinHigh(D7);
+    }
+    if (led_state.caps_lock) {
+        writePinLow(D6);
+    } else {
+        writePinHigh(D6);
+    }
+    if (led_state.scroll_lock) {
+        writePinLow(D4);
+    } else {
+        writePinHigh(D4);
+    }
+    return led_update_user(led_state);
```

Comment:
This should be:
```c
if (led_update_user(led_state)) {
    writePin(D7, !led_state.num_lock);
    writePin(D6, !led_state.caps_lock);
    writePin(D4, !led_state.scroll_lock);
}
return true;
```
which will allow user keymaps to override this for eg. layer indication.